CONTRIBUTING.md

```diff
@@ -34,6 +34,15 @@ to upgrade and/or update in order to protect their applications.
 
 For sensitive email communications, please use [our PGP key](http://framework.zend.com/zf-security-pgp-key.asc).
 
+## Documentation
+
+Documentation is in [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/),
```

Comment:
It does, but it renders all GFM syntax, and that's the point we want to get
across (particularly as the default rendering location is on github).
On Jun 12, 2015 2:56 PM, "colinodell" <notifications@github.com> wrote:

> In CONTRIBUTING.md
> <https://github.com/zendframework/zend-diactoros/pull/25#discussion_r32349219>
> :
>
> > @@ -34,6 +34,15 @@ to upgrade and/or update in order to protect their applications.
> >
> >  For sensitive email communications, please use [our PGP key](http://framework.zend.com/zf-security-pgp-key.asc).
> >
> > +## Documentation
> > +
> > +Documentation is in [GitHub Flavored Markdown](https://help.github.com/articles/github-flavored-markdown/),
>
> Doesn't bookdown use CommonMark (instead of GFM)?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/zendframework/zend-diactoros/pull/25/files#r32349219>.
>